pgmpy/tests/test_models/test_FunctionalBayesianNetwork.py

```diff
@@ -162,6 +170,146 @@ def test_simulate_different_distributions(self):
         self.assertTrue(np.all(samples["lognormal"] > 0))
         self.assertTrue(np.all(samples["gamma"] > 0))
 
+    def test_svi_fit_normal(self):
+        alpha = 0.23
+        x1 = np.random.normal(1, 2, size=5000)
+        x2 = np.random.normal((x1 * alpha) + 5, 1)
+        data = pd.DataFrame({"x1": x1, "x2": x2})
+
+        def x1_prior():
+            mu = pyro.param("x1_mu", torch.tensor(0.5))
+            sigma = pyro.param(
+                "x1_sigma",
+                torch.tensor(3.0),
+                constraint=torch.distributions.constraints.positive,
+            )
+            return dist.Normal(mu, sigma)
+
+        def x2_prior(parent):
+            mu = pyro.param("x2_mu", torch.tensor(3.0))
+            sigma = pyro.param(
+                "x2_sigma",
+                torch.tensor(2.0),
+                constraint=torch.distributions.constraints.positive,
+            )
+            alpha = pyro.param("x2_alpha", torch.tensor(1.0))
+            return dist.Normal(mu + (parent["x1"] * alpha), sigma)
+
+        cpd1 = FunctionalCPD("x1", fn=lambda _: x1_prior())
+        cpd2 = FunctionalCPD("x2", fn=lambda parent: x2_prior(parent), parents=["x1"])
+
+        self.model.add_cpds(cpd1, cpd2)
+
+        params = self.model.fit(data, method="SVI", learning_rate=0.05, num_steps=2000)
+
+        self.assertIn("x1_mu", params)
+        self.assertIn("x1_sigma", params)
+        self.assertIn("x2_mu", params)
+        self.assertIn("x2_sigma", params)
+        self.assertIn("x2_alpha", params)
+
+        self.assertAlmostEqual(params["x1_mu"].mean(), 1, delta=0.2)
+        self.assertAlmostEqual(params["x1_sigma"].mean(), 2, delta=0.2)
+        self.assertAlmostEqual(params["x2_mu"].mean(), 5, delta=0.2)
+        self.assertAlmostEqual(params["x2_sigma"].mean(), 1, delta=0.2)
+        self.assertAlmostEqual(params["x2_alpha"].mean(), 0.23, delta=0.2)
```

Comment:
delta is a bit too high for a simple model.  0.1 (or even lower) should be good.